Mathlib/Analysis/Convex/Combination.lean

```diff
@@ -551,3 +559,30 @@ theorem AffineBasis.convexHull_eq_nonneg_coord {ι : Type*} (b : AffineBasis ι
     rw [b.coord_apply_combination_of_mem hi hw₁] at hx
     exact hx
 #align affine_basis.convex_hull_eq_nonneg_coord AffineBasis.convexHull_eq_nonneg_coord
+
+variable {s t t₁ t₂ : Finset E}
+
+/-- Two simplices glue nicely if the union of their vertices is affine independent. -/
+lemma AffineIndependent.convexHull_inter (hs : AffineIndependent R ((↑) : s → E))
+    (ht₁ : t₁ ⊆ s) (ht₂ : t₂ ⊆ s) :
```

Comment:
Given the docstring, why not
```suggestion
/-- Two simplices glue nicely if the union of their vertices is affine independent. -/
lemma AffineIndependent.convexHull_inter (hs : AffineIndependent R ((↑) : (t₁ ∪ t₂) → E)) :
```
There should be some `mono` lemma that lets you recover your lemma via `union_subset`, right?

Reply:
This is a much less practical statement. Going from mine to yours is using `subset_union_left` + `subset_union_right` (discoverable) while going from yours to mine involves using `AffineIndependent.comp_embedding` + `Set.inclusion` + `Set.inclusion_injective` + `union_subset` (not very discoverable).

The translation from yours to mine would be simpler if we had an abbreviation for `AffineIndependent R ((↑) : s → E)`, but that feels out of scope for this PR.

Reply:
In that case, can we have both, with a comment on my one stating that often yours is easier to use?